### PR TITLE
docs: add Python versions supported

### PR DESCRIPTION
**Pull request by RobPasMue**
As title says

### CODE REVIEW

**Comment by RobPasMue** [comment]
image

File: doc/source/supported_versions.rst [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        axisFormat %Y-%m
+        Python 3.7  :done,   des1, 2018-06-27, 3y
+        Python 3.8  :done,   des2, 2019-10-14, 3y
+        Python 3.9  :active, des3, 2020-10-05, 3y
+        Python 3.10 :active, des4, 2021-10-04, 3y
+        Python 3.11 :active, des5, 2022-10-24, 3y
+        Python 3.12 :active, des6, 2023-10-02, 3y
+        Python 3.13 :        des7, 2024-10-01, 3y
+
+Below you can find a list of the Python versions supported by each PyAnsys metapackage release:
```
Comment (MaxJPRey) [inline doc/source/supported_versions.rst:36]: Do you think we should had a link toward the PyPI releases history of the metapackage?
https://pypi.org/project/pyansys/#history

I don't have any strong opinion.

**Review by MaxJPRey — APPROVED** [review]

File: doc/source/supported_versions.rst (hunk 1 above)
Comment (RobPasMue) [reply doc/source/supported_versions.rst:36]: I can do that, no problem at all.

**Review by SMoraisAnsys — APPROVED** [review]
LGTM

**Pull request merged**